### PR TITLE
10 add security types chart

### PR DESCRIPTION
**Pull request by MonikaFu**
Related to #10 

In this PR first chart of the meta report is added to the package. A data filtering function is added to the `data_processing.R` script and a plotting function is added to `plot_2diicharts.R`.

Expected output chart should look similar to this one:
image

@jdhoffa you might want to try this one out.

### CODE REVIEW

File: R/plot_2diicharts.R [hunk 1]
```diff
@@ -282,6 +282,64 @@ plot_techmix_chart <- function(data, plot_title = "", show_legend = TRUE,
...
+  }
+
+  p_bar <- ggplot(
+      data=data,
+      aes(fill = factor(.data$asset_type, levels = rev(bars_asset_type_specs$asset_type)),
+          x=factor(.data$investor_name, levels = rev(bars_labels_specs$investor_name)),
+          y=.data$share
+          )
+    ) +
+    theme_2dii_ggplot() +
```
Comment (maurolepore) [inline R/plot_2diicharts.R:324]: Maybe group all theming?

**Review by maurolepore — APPROVED** [review]
LGTM

File: R/data_processing.R [hunk 2]
```diff
@@ -97,3 +97,43 @@ filter_data_for_techmix_chart <- function(data, sector, years,
...
+#'
+#' @param data_total_portfolio dataframe in the shape of ".._total_portfolio.rda" dataset from PACTA analysis output in "30_Processed_Inputs" folder (dataframe)
+#' @param other_asset_types array of character strings that should be summed up as "Other" asset type (array of character strings; default = c("Funds","Others","Unclassifiable"))
+#'
+#' @description
+#' This function filters and aggregates one of PACTA analysis result files ".._total_portfolio.rda" from "30_Processed_Inputs" folder to form an input that can be used for plotting metareport security type coverage per investor type bar chart
+#'
+#' @export
+
+filter_data_for_metareport_security_type_chart <- function(data_total_portfolio,
```
[the remark this answers is not in the document]
Comment (MonikaFu) [reply R/data_processing.R:127]: good one. I will think of better names.

File: R/plot_2diicharts.R [hunk 3]
```diff
@@ -282,6 +282,64 @@ plot_techmix_chart <- function(data, plot_title = "", show_legend = TRUE,
...
+    scale_y_continuous(
+      labels = scales::percent_format(),
+      expand = c(0, 0),
+      sec.axis = dup_axis()
+    ) +
+    ylab("") +
+    xlab("") +
+    theme(axis.line.y = element_blank()) +
+    theme(axis.ticks.y = element_blank()) +
+    theme(legend.position = "bottom") +
```
[the remark this answers is not in the document]
Comment (MonikaFu) [reply R/plot_2diicharts.R:362]: that is done by `theme_2dii_ggplot()`. These elements are idiosyncratic to this chart.

**Pull request merged**